docs/ansible/roles/debops.phpipam/introduction.rst

```diff
@@ -0,0 +1,28 @@
+Introduction
```

Comment:
The introduction section should be moved to the `index.rst` file, check `debops.netbase` for a good example.

Reply:
done